doc/modules/changes/20220511_haoyuan

```diff
@@ -0,0 +1,4 @@
+New: The lower and upper limits of temperature of phase transitions.
+Phase transitions could be actived within a temperature range, and deactived outside.
+<br>
```

Comment:
There is still an unresolved comment by jdannberg about this changelog entry. For reference, this is the comment: https://github.com/geodynamics/aspect/pull/4591/files#r874136668

Reply:
Hi Rene, I have merged your PR and fixed the tests with the 3 new prm files.